apps/src/templates/progress/ProgressLessonTeacherInfo.jsx

```diff
@@ -33,7 +33,7 @@ class ProgressLessonTeacherInfo extends React.Component {
     unitName: PropTypes.string.isRequired,
     hasNoSections: PropTypes.bool.isRequired,
     toggleHiddenLesson: PropTypes.func.isRequired,
-    lockableAuthorized: PropTypes.bool.isRequired
+    lockableAuthorized: PropTypes.bool
```

Comment:
Why is this prop no longer required?

Reply:
It is now null before data loads instead of defaulting to `false`